### PR TITLE
Bulk views for users: state, password and delete

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
TODO: Write some tests.

**Comment by FinnStutzenstein** [comment]
@GabrielInTheWorld @tsiegleauq Please test

File: client/src/app/site/users/components/user-list/user-list.component.ts [hunk 1]
```diff
@@ -395,18 +375,15 @@ export class UserListComponent extends BaseListViewComponent<ViewUser> implement
...
                 this.translate.instant(
                     'Note: Your own password was not changed. Please use the password change dialog instead.'
                 )
             );
         }
-        for (const user of this.selectedRows.filter(u => u.user.id !== this.operator.user.id)) {
-            const password = this.repo.getRandomPassword();
-            this.repo.resetPassword(user, password, true);
-        }
+        this.repo.bulkResetPassword(this.selectedRows).then(null, this.raiseError);
```
Comment (tsiegleauq) [inline client/src/app/site/users/components/user-list/user-list.component.ts]: Show prompt here

File: client/src/app/core/repositories/users/user-repository.service.ts [hunk 2]
```diff
@@ -118,6 +118,21 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User, UserTi
         return this.translate.instant(plural ? 'Participants' : 'Participant');
     };
 
+    /**
+     * Generates a random password
+     *
+     * @param length The length of the password to generate
+     * @returns a random password
+     */
+    public getRandomPassword(length: number = 8): string {
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/users/user-repository.service.ts]: To have the same as Django, use 10 instead of 8

**Comment by FinnStutzenstein** [comment]
> * idea: Password rest should not manipulate the initial PW, but rather send an e-mail after generating the new PW

Can be discussed later; not for this PR.

File: client/src/app/site/users/components/user-detail/user-detail.component.ts [hunk 3]
```diff
@@ -437,11 +437,11 @@ export class UserDetailComponent extends BaseViewComponent implements OnInit {
     /**
      * (Re)- send an invitation email for this user after confirmation
      */
-    public async sendInvitationEmail(): Promise<void> {
+    public async bulkSendInvitationEmail(): Promise<void> {
```
Comment (tsiegleauq) [inline client/src/app/site/users/components/user-detail/user-detail.component.ts]: User detail operations are not "bulk". Probably too much renaming.

**Pull request merged**